api/desecapi/tests/test_dyndns12update.py

```diff
@@ -142,6 +142,13 @@ def test_identification_by_token(self):
         self.assertEqual(response.data, 'good')
         self.assertIP(ipv4='10.5.5.6')
 
+    def test_identification_by_email(self):
+        self.client.set_credentials_basic_auth(self.owner.email, self.token.key)
```

Comment:
This may break tests if executed in a different test set or different order. I propose to overwrite the client's credentials in the `setUp()` method

Reply:
As far as I can see, the `set_credentials_basic_auth` [is actually run on the client inside `setUp`](https://github.com/desec-io/desec-stack/blob/aee5777d960ee71176a05268d42a6c8ae36a501f/api/desecapi/tests/base.py#L748) of a parent class. So we're good -- or am I missing some aspect here?

Reply:
ah, woops, I didn't catch that